### DISCUSSION TITLE
Pouvoir voir d'autres joueurs sur le !map

### DISCUSSION THREAD

**Discussion by SomniumWed** [Crownicles/Crownicles#1078] [Suggestions fermées]
Je propose de pouvoir voir d'autres personnes que nous sur la map à l'aide d'une mention/id d'utilisateur/rang dans !t/...
Par exemple le curseur rouge indique notre position et un curseur d'une autre couleur indique la position de la personne mentionné

**Comment by BastLast** [comment]
Bonjour c'est quasiment impossible à coder si on veut garder des performances correct avec la machine actuelle sur laquelle tourne le bot.

Il nous est malheureusement impossible de réaliser du traitement d'image sans augmenter drastiquement les performances et pour des raisons évidentes de coût je ne pense pas que cela sera réalisé

**Reply by BastLast** [reply]
Après moyennant un petit don de 1500 € moi je suis chaud pour faire tourner cette fonctionnalité pendant quelques mois. ( juste pour donner un ordre d'idée de à quel point ça coûte cher de avoir un serveur beaucoup plus puissant)

**Reply by BastLast** [reply]
Par contre ce qui serait faisable et qui peut éventuellement faire l'objet d'une nouvelle suggestion ça serait de juste ajouter la possibilité dans la commande map de donner une mention et d'avoir la carte tu aimes quel serait afficher si le joueur mentionné avait fait la commande

**Reply by BastLast** [reply]
Telle qu'elle serait *

**Reply by BastLast** [reply]
Affichée * la dictée vocale c'est de la merde
